opennebula/defaults.yaml

```diff
@@ -2,50 +2,42 @@ kernel:
   Linux:
     oneadmin:
       home: /var/lib/one
-      optional_groups:
-        - kvm
-        - libvirt
-        - ssl-cert
       config:
         manage:
-          - ec2_auth
-          - occi_auth
+#          - ec2_auth
```

Comment:
you can override data that is listed in the defaults.yaml, please keep these comments only if we really need them